cruise/README.md

```diff
@@ -0,0 +1,7 @@
+# Cruise CSV
+
+Please following these guidelines:
+* Keep cruises in chronological order
+* Put the Array Name first in the Notes column
+* Be sure to include a time (e.g., "T00:00:00") as well as date in the DateTime columns
```

Comment:
@snwhite Hi Sheri, that single cruise without any times was my fault.  I thought I had added times, but I guess they didn't make it in before I submitted the pull request. Once your changes go through, I will also update that same one to have Endurance at the beginning of the Notes. -Stuart